docs/specs/Broker-CI.md

```diff
@@ -0,0 +1,103 @@
+# Broker CI
```

Comment:
Broker Continuous Integration

I'd spell it out the first time. I know it's a common term but it's good to state it at least once.